### PR TITLE
Remove events from Otel Kubernetes General Dashboard

### PR DESCRIPTION
**Pull request by gizas**
## Proposed commit message

WHAT: Removing the events from overview dashboard
WHY: https://github.com/elastic/opentelemetry-dev/issues/469

This version includes some resizing of visualisations in ordeto make UX more friendly to smaller display analysis

### CODE REVIEW

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json [hunk 1]
```diff
@@ -4017,727 +4017,121 @@
...
-                    "x": 23,
-                    "y": 45
-                },
-                "panelIndex": "7b762c59-2643-4200-8c76-ece0937e11e7",
-                "type": "lens"
             }
         ],
         "timeRestore": false,
-        "title": "[OTEL][Metrics Kubernetes] Cluster Overview",
+        "title": "[OTEL][Metrics Kubernetes] Cluster Overview (1)",
```
Comment (tetianakravchenko) [inline packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json:4031]: please fix the name

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json [hunk 2]
```diff
@@ -4017,727 +4017,121 @@
...
         {
             "id": "metrics-*",
-            "name": "67779719-91a4-4a68-b7fe-7bc7eef837af:indexpattern-datasource-layer-5cd693aa-6e5b-43de-b3c5-3d353a0644a3",
+            "name": "6b564db2-f6f7-4bf4-8fab-534b6bb47c55:indexpattern-datasource-layer-5cd693aa-6e5b-43de-b3c5-3d353a0644a3",
             "type": "index-pattern"
         },
+        {
+            "id": "kubernetes_otel-fleet-shared-tag-kubernetes_otel-dfd505f6-fc02-5c53-adc7-148425d7972c-default",
+            "name": "tag-ref-fleet-shared-tag-kubernetes_otel-dfd505f6-fc02-5c53-adc7-148425d7972c-default",
+            "type": "tag"
```
Comment (tetianakravchenko) [inline packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json:4133]: please remove `tag`, doesn't it cause the failure of the `elastic-package build` ?

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json (hunk 1 above)
Comment (gizas) [reply packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json]: Done !

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json (hunk 2 above)
Comment (gizas) [reply packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json]: Indeed removed

**Review by tetianakravchenko — APPROVED** [review]

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json [hunk 3]
```diff
@@ -907,56 +900,16 @@
...
-                                "query": {
-                                    "language": "kuery",
-                                    "query": ""
-                                }
-                            }
-                        },
-                        "description": "",
-                        "params": {
-                            "fontSize": 12,
-                            "markdown": "### Node overview\n***",
```
[the remark this answers is not in the document]
Comment (gizas) [reply packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-cluster-overview.json:942]: Seems like a false error, Node overview is there

Screenshot 2024-10-17 at 3 56 50 PM

**Pull request merged**